cli/src/commit_templater.rs

```diff
@@ -301,6 +359,15 @@ fn build_commit_keyword_opt<'repo>(
         "committer" => {
             language.wrap_signature(wrap_fn(property, |commit| commit.committer().clone()))
         }
+        "signature" => {
+            language.wrap_commit_signature(wrap_fn(property, |commit| {
+                match commit.verification() {
+                    Ok(Some(v)) => CommitSignature::Present(v),
+                    Err(SignError::InvalidSignatureFormat) => CommitSignature::Invalid,
+                    _ => CommitSignature::Absent, // todo don't ignore verification errors
```

Comment:
nit: write it as "TODO: " to make it easier for tools to recognize (i think)

Reply:
for me vscode highlights any comment with todo in any case (I don't think I've configured that?.), so all my todos are like this, sorry :)